Opis-innowacji-aplikacje.gov.pl-EZD.rst

```diff
@@ -0,0 +1,452 @@
+   instytucja centralna),
+-  obsłużenie ewentualnych uwag do spisów zdawczo-odbiorczych,
+-  poinformowanie o zaakceptowaniu spisu zdawczo-odbiorczego
+-  W przypadku dokumentacji papierowej moduł powinien obsługiwać
+   wypożyczenia fizycznych kopii dokumentów zarówno po stronie
+   pracowników instytucji, jak i pracowników Archiwum Zakładowego
+
+**19. Paczka migracyjna**
+=========================
+
+Paczka migracyjna jest częścią systemu, która wpływa na projektowanie
+jedynie (i aż) w zakresie zbieranych w systemie metadanych. W związku z
+tym wykaz metadanych/pól będzie zawsze powiązany z aktualnym stanem
+wiedzy na temat paczki migracyjnej.
+
+**24. Procesowość**
+===================
+
+Ten moduł pozwala (uwzględniając aktualną strukturę organizacyjną)
+ustawiać procesy w instytucji. Procesy powinny być możliwe do podglądu w
+BPMN. Jednak sam interfejs tworzenia procesów należy dostosować do
+oczekiwań użytkowników administracji publicznej. Być może forma pytań o
+relacje będzie lepszym rozwiązaniem na etapie konfiguracji.
+
+**#innowacja Generator procesów wyłapuje potencjalne pętle w procesach
+już na etapie ich wprowadzania**
+
+**26. Baza adresatów**
+======================
+
+Baza adresatów to część systemu, która jest wspólna na pewno dla
+wszystkich użytkowników w ramach jednej instytucji-instancji. System
+powinien wymuszać korzystanie w pierwszej kolejności z danych w bazie
+adresatów. W przypadku braku poprawności zachęcać do ich aktualizacji
+(umożliwiając zamieszczenie wewnętrznego komentarza na temat dokonanej
+zmiany). Własne bazy adresatów instytucji powinny być ograniczone do
+minimum, korzystając maksymalnie z ogólnodostępnych baz takich jak np.
+REGON.
+
+**#innowacja System stosuje hierarchię źródeł danych adresowych co ma
+prowadzić do pozyskiwania najaktualniejszych danych, które nie dublują
+istniejących baz**
+
+**27. Metadane, słowniki, wzory**
+=================================
+
+**#innowacja Geotagowanie spraw i obiektów**
+
+Geotagowanie będziem możliwe w miejscu
+
+::
+
+    InnyTemat ::= inne klucz Klucz klucz wartosc WartośćKlucza wartosc inne 
+
+Długość i szerokość geograficzna to będą 2 pola *InnyTemat*
+
+--------------
+
+Zestaw i formaty podstawowych metadanych w systemie będzie określany z
+góry wymogami paczki migracyjnej. Jednak musimy założyć sytuację, gdy
+instytucje będą chciały dodawać dodatkowe metadane. W tym module musi
+być miejsce na określenie ich listy, jakich obiektów w systemie dotyczą,
+jaki mają format.
+
+Kolejny temat to słowniki w ramach instytucji (np. teleadresowy, kodow
+pocztowych). Minimum jest funkcjonowanie wspólnego słownika w ramach
+jednej instytucji. Powinniśmy założyć również możliwość skorzystania ze
+słownika współdzielonego przez inne instytucje lub będącego
+słownikiem/bazą centralną (np. REGON).
+
+Wzory dokumentów to kolejny ważny element zapewnienia spójności w ramach
+urzędu, ale także oszczędzenia pracownikom i pracowniczkom sprawdzania

```

Comment:
Wg słownika PWN „pracownik” to „osoba zatrudniona w jakimś zakładzie pracy”, więc „pracowniczka” już się w tym zawiera.